data/conf/infrasim.conf

```diff
@@ -1,59 +0,0 @@
-#This file is used for InfraSIM OpenIPMI configuration file
-
-[main]
-#Supported vnode names:
```

Comment:
More detailed comments required:
virtual server node personality sounds better than vnode name; explanations like specifying node personality applies particular emulation data for BMC and SMBIOS table. For example, by default node=quanta_d51, it means the virtual server node has similar BMC sensors, SDRs and configrations and most behaviors. And the output from running dmidecode is the same as the output caputred from a physical Quanta D51 server.